src/components/button/examples/demo/CTA.vue

```diff
@@ -14,7 +14,6 @@
         :key="index2"
         :size="button.size"
         :full-width="button.fullWidth"
-        :disabled="button.disabled"
```

Comment:
This is not a valid attribute value (nor is it needed for this example). I think the JSX compilation must have ignored it